docs/releases/v13/version-v13.0.0.md

```diff
@@ -24,13 +24,11 @@ Related [new doc page](../../operate/customize/vocabularies/names.md#using-orcid
 
 ### Search improvements
 
-Addition of a [`suggest` API](../../reference/rest_api_suggest.md).
+Both user and record search have been enhanced to return more accurate results for common names/titles, partial matches (even with typos) and names/titles with accents or diacritics.
+	
+Creators, affiliations and funders autocomplete search has been enhanced so suggestions appear faster and better match what you type.
```

Comment:
minor:
```suggestion
Creators, affiliations and funders autocomplete search has been enhanced so that suggestions appear faster and better match what you type.
```